synapse/lib/stormlib/imap.py

```diff
@@ -72,7 +394,7 @@ def getObjLocals(self):
             'connect': self.connect,
         }
 
-    async def connect(self, host, port=993, timeout=30, ssl=True, ssl_verify=True):
+    async def connect(self, host, port=imaplib.IMAP4_SSL_PORT, timeout=30, ssl=True, ssl_verify=True):
```

Comment:
note: `imaplib.IMAP4_SSL_PORT` is not a documented constant.

Reply:
It's been there for 24 years. I think we're safe.
https://github.com/python/cpython/blame/main/Lib/imaplib.py#L44